std/range/primitives.d

```diff
@@ -2004,11 +2004,12 @@ ElementType!R moveAt(R)(R r, size_t i)
 
 /**
 Implements the range interface primitive $(D empty) for built-in
-arrays. Due to the fact that nonmember functions can be called with
+arrays and associative arrays. Due to the fact that nonmember functions can be called with
 the first argument using the dot notation, $(D array.empty) is
 equivalent to $(D empty(array)).
```

Comment:
You need to rewrite this doc because the function is now much more general. "Implements the `empty` range primitive for types that obey `hasLength` (please link) and for narrow strings."